Makefile

```diff
@@ -5,10 +5,20 @@ help:
 	@echo "Please use 'make <target>' where <target> is one of:"
 	@echo "  help                          to show this message"
 	@echo "  build-container               to build the container image for the quipucords UI"
+	@echo "  bump-version <version>        to bump quipucords UI version"
 
 all: help
 
 build-container:
 	podman build \
 		-t $(QUIPUCORDS_UI_CONTAINER_TAG) --ulimit nofile=4096:4096 .
 
+bump-version:
+ifeq ($(version),)
+$(error Variable "version" is not set)
+endif
+	$(eval MINOR=$(shell echo $(version) | sed 's/\([0-9]*\.[0-9]*\).*$$/\1/'))
+	@echo "Bumping version to $(version)"
+	sed -i 's/^\(LABEL version=\).*$$/\1$(version)/' Containerfile
+	sed -i 's/^\(LABEL version_minor=\).*$$/\1$(MINOR)/' Containerfile
+	npm run release -- --override $(version)
```

Comment:
I believe this command generates a commit, but something else still needs to actually push it to the remote. Is anything pushing here? Or do you know if that's irrelevant for this process?

Reply:
oh, does npm release generate a commit? didn't knew that.

Oh crap, it needs to be ajusted then to git add Containerfile.

Answering your question, it is not pushing anything.

Reply:
maybe I should update the release script instead of doing this here 🤔

Reply:
But I'd still add as a Makefile target because my intention is to have this process streamlined for all repos

Reply:
mmm this is the changelog script, an external dep
https://github.com/cdcabrera/changelog-light

Reply:
I'm not seeing a way to add more files to the commit it will create. I think we will need to run it with `--commit=false` and start commiting the changes manually

Reply:
I completely altered the approach here, so I'm going to mark this discussion as resolved.
Feel free to reopen if you I missed anything, @infinitewarp